src/hooks/useGeneratorColor/utils/randomIndex.ts

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-bitwise */
```

Comment:
Can we write any tests for utils?

Reply:
yes, it's important to test two utilities: getHue and randomIndex